utils/conversions.ts

```diff
@@ -82,23 +84,31 @@ export const toApiEligibilityCheck = (
  */
 export const toModelBonusActivation = (
   apiObj: ApiBonusActivation
-): Either<t.Errors, BonusActivation> => {
+): Either<t.Errors, BonusActivationWithFamilyUID> => {
   const camelCasedUntypedObj = renameObjectKeys(apiObj, k =>
     snakeCaseToCamelCase(k)
   );
-  return BonusActivation.decode(camelCasedUntypedObj);
+  return BonusActivation.decode(camelCasedUntypedObj).chain(base =>
+    BonusActivationWithFamilyUID.decode({
```

Comment:
just call 

```
BonusActivationWithFamilyUID.decode({ ... camelCasedUntypedObj, familyUID: generateFamilyUID(base.dsuRequest.familyMembers) })
```

Reply:
I can't, because `generateFamilyUID` expects a domain type, and this is obtained by the first decode (the `base` object)